src/features/breadcrumbs/BreadcrumbsPanel.tsx

```diff
@@ -19,6 +20,7 @@ const noBreadcrumbsOption = {
 
 export function BreadcrumbsPanel() {
   const [items] = useAtom(breadcrumbsItemsAtom);
+  const [currentMapPosition] = useAtom(currentMapPositionAtom);
```

Comment:
For me its logically not right that BreadcrumbsPanel component depends on the `currentMapPositionAtom`, i think that  it should only depend on items, and dependency on `currentMapPositionAtom` should be hided somewhere intrernally. What do you think?

Reply:
Technically, the logic of the component still relied on the same subscription to the map position (you can't just pass items to BreadcrumbsPanel, which is okay), just more obvious this time. I do agree that this is still not perfect, because it's subscription to two atoms and a debounce function in between them. Feels like debounce + subscribing to position could be a hook

Reply:
I've created a PR to address this issue, can you take a look please? https://github.com/konturio/disaster-ninja-fe/pull/922